tests/end-to-end/verify_diff.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+"""Test different output files against expected files."""
+
+import argparse
+
+import sys
+
+import torch
+
+EPS = 0.0001
+
+
+def parse_args():
+    """Parse command line arguments.
+
+    Return:
+        args : parsed argument object.
+
+    """
+    parser = argparse.ArgumentParser(
+        "Utils scripts for verifying identicality of different files.")
+    parser.add_argument('--result_path', type=str,
+                        help='Path to generated result file.')
+    parser.add_argument('--expected_path', type=str,
+                        help='Path to expected result file.')
+    parser.add_argument('--format', type=str,
```

Comment:
I don't see a verify function for `bw`. is that not needed anymore?

Reply:
removed, good catch.